android/gradle.properties

```diff
@@ -19,9 +18,8 @@ org.gradle.jvmargs=-Xmx2560m -XX:+UseParallelGC
 org.gradle.parallel=false
 org.gradle.daemon=false
 
-# Jul 12 2019 - To solve Manifest merger failed Attribute application@appComponentFactory value=(android.support.v4.app.CoreComponentFactory) from [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91 is also present at [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
Jetifier is disabled now. I also removed the comment above that no longer seemed necessary.